docker-compose.yml

```diff
@@ -105,6 +105,9 @@ services:
   web:
     <<: *app
     command: bash -c "/bin/docker-entrypoint.sh"
+    environment:
+      - VIRTUAL_HOST=nims.docker
```

Comment:
These could be inside the .env file (the .env.template file)

Reply:
yes, those are only because I use dory (which is ace btw) so I will move them